astropy/timeseries/periodograms/lombscargle/implementations/fastchi2_impl.py

```diff
@@ -21,17 +21,17 @@ def lombscargle_fastchi2(t, y, dy, f0, df, Nf, normalization='standard',
         broadcastable to the same shape.
     f0, df, Nf : (float, float, int)
         parameters describing the frequency grid, f = f0 + df * arange(Nf).
-    normalization : string (optional, default='standard')
+    normalization : str, optional
```

Comment:
keep the defaults here, and in the other files, too.

Reply:
In general do we want to duplicate the default in the description if it's also in the function signature? I agree for cases where the signature shows None, but otherwise we open ourselves up for having inconsistent docstrings compared to the function signature?

Reply:
Hmm, good question. My biggest concern is that we don't always have the signature due to the class inheritances, but in many cases do manually inherit the docstrings, etc.

Reply:
Ok, well I can add back the defaults - I'm just a little worried that we might not be very consistent in when we do that and not.

Reply:
it might be just my misconception, but so far I think we list them. However I fully agree with you that it would be much nicer to rely on the signature only, so in fact I'm OK for keeping this as is, as a first step to clean up stuff.
(To show how bike shedding commenting this here was: I don't even think this particular docstring is available anywhere in the API docs, the method functions are hidden away and their kwargs can be adjusted by `method_kwds`. In fact it would be nice to somehow expose what kwargs are available, but that's well beyond the scope here 🤷‍♀

Reply:
Yeah, let's just leave off the defaults in the docstring. Should be sufficient in the function signature.

Reply:
Ok I'll not add back the defaults for now.